libs/utils/android/benchmark.py

```diff
@@ -116,7 +116,7 @@ def _parseCommandLine(self):
 
         # Test customization
         parser.add_argument('--results-dir', type=str,
-                default=None,
+                default=self.__class__.__name__,
```

Comment:
The `<DATE_TIME>` is a default if you do not specify a custom path, and the results folder becomes quite confusing and messy once you run it over time. You cannot even know for sure what a `<DATA_TIME>` folder contains exactly.

Should we not fix instead our synthetics to properly generate results using a more clean folder structure, e.g. something like `results/synthetics/<TEST_NAME>`... and maybe we can also append a `<DATE_TIME>` to this base path in case we really want to keep an history of different executions?